paasta_tools/paasta_cli/cmds/emergency_scale.py

```diff
@@ -0,0 +1,80 @@
+#!/usr/bin/env python
+# Copyright 2015 Yelp Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from service_configuration_lib import DEFAULT_SOA_DIR
+
+from paasta_tools.paasta_cli.utils import execute_paasta_serviceinit_on_remote_master
+from paasta_tools.paasta_cli.utils import figure_out_service_name
+from paasta_tools.paasta_cli.utils import lazy_choices_completer
+from paasta_tools.paasta_cli.utils import list_services
+from paasta_tools.paasta_cli.utils import list_instances
+from paasta_tools.utils import compose_job_id
+from paasta_tools.utils import list_clusters
+
+
+def add_subparser(subparsers):
+    status_parser = subparsers.add_parser(
+        'emergency-scale',
+        description="Scale a PaaSTA service instance",
+        help=("Scale a PaaSTA service instance by scaling it up or down to n instances (for Marathon apps)"))
+    status_parser.add_argument(
+        '-s', '--service',
+        help="Service that you want to scale. Like 'example_service'.",
+    ).completer = lazy_choices_completer(list_services)
+    status_parser.add_argument(
+        '-i', '--instance',
+        help="Instance of the service that you want to scale. Like 'main' or 'canary'.",
+        required=True,
+    ).completer = lazy_choices_completer(list_instances)
+    status_parser.add_argument(
+        '-c', '--cluster',
+        help="The PaaSTA cluster that has the service instance you want to scale. Like 'norcal-prod'.",
+        required=True,
+    ).completer = lazy_choices_completer(list_clusters)
+    status_parser.add_argument(
+        '-a', '--appid',
```

Comment:
No, this is a necessary argument to allow people to kill *specific* apps (or now, scale?) in emergency situations. Specifically in the case of a bounce gone wrong, where bounces do not progress because they are stuck.